src/jit/simd.h

```diff
@@ -5,6 +5,42 @@
 #ifndef _SIMD_H_
 #define _SIMD_H_
 
+// Underlying hardware information
+// This type is used to contorl
```

Comment:
typo: contorl => control

Reply:
Done. Thanks for pointing out. Also modified comments of `InstructionSet`.